themes/digital.gov/layouts/partials/core/footer.html

```diff
@@ -48,11 +52,36 @@ <h4 class="usa-footer__primary-link">About</h4>
             <section class="usa-footer__primary-content usa-footer__primary-content--collapsible">
               <h4 class="usa-footer__primary-link">Social</h4>
               <ul class="usa-list usa-list--unstyled social-links">
-                <li class="usa-footer__secondary-link"><i class="fab fa-twitter"></i> <a class="text-ink hover:text-ink text-no-underline hover:text-underline" href="https://twitter.com/digital_gov">Twitter</a></li>
-                <li class="usa-footer__secondary-link"><i class="fab fa-facebook-f"></i> <a class="text-ink hover:text-ink text-no-underline hover:text-underline" href="https://www.facebook.com/digitalgov/">Facebook</a></li>
-                <li class="usa-footer__secondary-link"><i class="fab fa-youtube"></i> <a class="text-ink hover:text-ink text-no-underline hover:text-underline" href="https://youtube.com/digitalgov">YouTube</a></li>
-                <li class="usa-footer__secondary-link"><i class="fas fa-rss"></i> <a class="text-ink hover:text-ink text-no-underline hover:text-underline" href="{{- "/" | relURL -}}index.xml">RSS</a></li>
-                <li class="usa-footer__secondary-link"><i class="far fa-envelope"></i> <a class="text-ink hover:text-ink text-no-underline hover:text-underline" href="mailto:digitalgov@gsa.gov">Email us</a></li>
+                <li class="usa-footer__secondary-link">
+                  <svg class="dg-icon dg-icon__margin-bottom-4" aria-hidden="true" focusable="false" role="img">
+                    <use xlink:href="/uswds/img/sprite.svg#twitter"></use>
```

Comment:
These icons work locally, but not on federalist. Try using the base url variable `{{- .Site.BaseURL -}}` for all environments.

Reply:
I've updated all svg paths to use `{{- .Site.BaseURL -}}`, thanks for catching this!